lib/apartment/adapters/abstract_adapter.rb

```diff
@@ -30,8 +30,11 @@ def create(tenant)
             config = config_for(tenant)
             difference = current_difference_from(config)
 
-            if difference[:host]
-              connection_switch(config, without_keys: [:database, :schema_search_path])
+            if difference[:host] || difference[:port]
+              default_db = "postgres"
+              temp_config = config.dup
+              temp_config[:database] = default_db
+              connection_switch!(temp_config)
```

Comment:
That doesn't apply to us since we never set (or change) the schema search path. That functionality is for having multiple tenants on the same database separated with a different schema.